docs/integrations/sources/iterable.md

```diff
@@ -72,10 +72,15 @@ The Iterable source connector supports the following [sync modes](https://docs.a
 * [CustomEvent](https://api.iterable.com/api/docs#export_exportDataJson) \(Incremental\)
 * [HostedUnsubscribeClick](https://api.iterable.com/api/docs#export_exportDataJson) \(Incremental\)
 
+## Additional notes
+
+[List Users](https://api.iterable.com/api/docs#lists_getLists_0) Stream when meet `500 - Generic Error` will skip a broken slice and keep going with the next one. This is related to unexpected failures when trying to get users list for specific list ids.
```

Comment:
Let's link to this issue too https://github.com/airbytehq/airbyte/issues/24968

Reply:
done